opencti-platform/opencti-front/src/private/components/settings/groups/GroupEditionOverview.tsx

```diff
@@ -95,6 +105,40 @@ const GroupEditionOverviewComponent: FunctionComponent<GroupEditionOverviewCompo
 
   const editor = useFormEditor(group as unknown as GenericData, false, queries, groupValidator);
 
+  const handleSubmitField = (name: string, value: string) => {
+    groupValidator
+      .validateAt(name, { [name]: value })
+      .then(() => {
+        if (name === 'group_confidence_level') {
+          if (group.group_confidence_level) {
+            commitFieldPatch({
+              variables: {
+                id: group.id,
+                input: {
+                  key: 'group_confidence_level',
+                  object_path: '/group_confidence_level/max_confidence',
+                  value: parseInt(value, 10),
+                },
+              },
+            });
+          } else {
+            commitFieldPatch({
+              variables: {
+                id: group.id,
+                input: {
+                  key: 'group_confidence_level',
+                  value: {
+                    max_confidence: parseInt(value, 10),
+                    overrides: [],
+                  },
+                },
+              },
+            });
+          }
```

Comment:
the else statement could work in both cases (existing confidence or not) right now, but we will implement soon the overrides definition and edition (6.1). By then we'll need to patch with `object_path` anyway.